build-container-image.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
```

Comment:
It would be great to document the script execution briefly in our readme, so that users can quickly get up and running.